python/test/unit/language/test_core.py

```diff
@@ -1493,8 +1493,8 @@ def serialized_add(data, Lock, SEM: tl.constexpr):
 
     Lock = torch.zeros((1, ), device=device, dtype=torch.int32)
     data = torch.zeros((128, ), device=device, dtype=torch.float32)
-    ref = torch.full((128, ), 64.0)
-    h = serialized_add[(64, )](data, Lock, SEM=sem, num_ctas=num_ctas)
+    ref = torch.full((128, ), 2000.0)
+    h = serialized_add[(2000, )](data, Lock, SEM=sem, num_ctas=num_ctas)
```

Comment:
Is this testing enough or do we need more testing for the new feature ?

Reply:
I think we already have the test `test_atomic_rmw` in `test_core.py` to cover the different memory oderings.